Assets/Scripts/Layers/LayerManager.cs

```diff
@@ -174,6 +175,11 @@ void SetContextMenuPosition(Vector2 position)
 
         private void Update()
         {
+            if (Keyboard.current.deleteKey.wasPressedThisFrame)
```

Comment:
Had dit niet beter een Action in het InputSystem kunnen zijn waar we op binden? Dat maakt het consistent met hoe het systeem bedoeld is, en ook mogelijk om button remapping te doen of het te binden op andere controller acties. Als je dit in een losse PR wil aanbieden en deze mergen dan klinkt dat goed; maar laten we het dan niet vergeten :)

Reply:
Oh, en het scheelt ook spul in de Update methode; omdat je dan direct op events bind

Reply:
Volgens mij kan ik onze standaard action map niet aanpassen omdat we de default versie gebruiken, laten we hier morgen even naar kijken. ik zet er een todo bij